docs/src/components/LoadingIndicator/Basic.tsx

```diff
@@ -0,0 +1,39 @@
+import {LoadingIndicator, ThemeProvider, ThemeType, useTheme} from 'dooboo-ui';
+
+import type {FC} from 'react';
+import {View} from 'react-native';
+
+export const Component: FC = () => {
+  const {theme} = useTheme();
+
+  return (
+    <View>
```

Comment:
I suggest you to wrap things with `Fragment(<></>)` if you are not giving props to `View`.